Makefile

```diff
@@ -35,9 +35,8 @@ endif
 OS := $(shell uname)
 ifeq ($(OS),Darwin)
 install-spellchecker-deps: ## Install MacOS dependencies for spellchecker
-	brew install enchant
-	pip install sphinxcontrib-spelling
-else
+	brew install enchant \
```

Comment:
looks like this caused the CI to break @jack1902

Reply:
oops, fixed now!

Reply:
@parano think you meant @jackyzha0